packages/contracts/deploy-config/mainnet.json

```diff
@@ -43,7 +43,7 @@
   "governorVotingPeriodBlocks": 21600,
   "governorProposalThreshold": 1,
   "governorVotesQuorumFractionPercent": 51,
-  "timeLockMinDelaySeconds": 2592000,
+  "timeLockMinDelaySeconds": 604800,
```

Comment:
How about separate this into another PR? It's related to changing timelock delay from 30 days to 7 days.

Reply:
OK, I'll explicitly separate the PR.